modules/generating-icsp-object-scoped-to-a-registry.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc:
+
+[id="generating-icsp-object-scoped-to-a-registry_{context}"]
+= Widening the scope of the mirror image catalog to reduce the frequency of cluster node reboots
+
+You can scope the mirrored image catalog at the repository level or the wider registry level. A widely scoped `ImageContentSourcePolicy` resource reduces the number of times the nodes need to reboot in response to changes to the resource.
+
+To widen the scope of the mirror image catalog in the `ImageContentSourcePolicy` resource, do the following:
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in as a user with `cluster-admin` privileges.
+* Configure a mirrored image catalog repository for use in your disconnected cluster.
+
+.Procedure
+
+. Run the following command, specifying values for `<local_registery>` and `<pull_secret_file>`:
++
+[source,terminal]
+----
+$ oc adm catalog mirror <local_registery>/olm/redhat-operators:v1 <local_registery> -a <pull_secret_file> --icsp-scope=registry
+----
++
+where:
+
+<local_registery>:: is the local registry you have configured for your disconnected cluster, for example `local.registry:5000`.
+<pull_secret_file>:: is the `registry.redhat.io` pull secret in `.json` file format downloaded from the link:https://cloud.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {cloud-redhat-com} site.
++
+The `oc adm catalog mirror` command creates a `/redhat-operator-index-manifests` directory and generates `imageContentSourcePolicy.yaml` and `mapping.txt` files.
+
+. Apply the new `ImageContentSourcePolicy` resource to the cluster:
++
+[source,terminal]
+----
+$ oc apply -f redhat-operator-index-manifests/imageContentSourcePolicy.yaml
+----
+
+.Verification
+* Verify that `oc apply` successfully applied the change to `ImageContentSourcePolicy`:
++
+[source,terminal]
+----
+$ oc get ImageContentSourcePolicy -o yaml
+----
++
+.Expected output
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: redhat-operators
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - local.registry:5000
+    source: registry.redhat.io
+----
+
+After you update the `ImageContentSourcePolicy` resource, OpenShift deploys the new settings to each node and the cluster starts using the mirrored repository for requests to the source repository.
```

Comment:
Please replace `OpenShift` with either {product-title} or whatever does the copying of the files.  I appreciate your effort to avoid passive voice.